api/src/controllers/team.js

```diff
@@ -221,9 +219,10 @@ async function deployToGHPages(req, res, next) {
 
   const body = {
     ghUsername: ghUser,
```

Comment:
Consider calling the variable `ghUsername` as well for consistency, seeing as we don't use it for anything else.